src/components/pages/profile/profile-description-list.component.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import PropTypes from 'prop-types';
+
+const ProfileDescriptionList = ({ userData }) => {
+  const userDataKeys = Object.keys(userData);
+  const descList = userDataKeys.map((key,index) => (
+    <ListItemText
+      primary={key}
+      secondary={userData[key]}
+      key={key}
+    />
+  ));
+  return (
+    <List>
+      <ListItem>
+        {descList}
+      </ListItem>
+    </List>
+  );
+};
```

Comment:
This whole component could be more simple and the List definition is not so correct.
```javascript
const ProfileDescriptionList = ({ userData }) => (
  <List>
    {Object.keys(userData).map(key => (
      <ListItem key={key}>
        <ListItemText
          primary={key}
          secondary={userData[key]}
        />
      </ListItem>
    ))}
  </List>
);
```